main.py

```diff
@@ -11,19 +11,19 @@ def main():
         application.run()
         sys.exit(0)
     except UserException as err:
-        message = 'User exception: %s' % err
+        message = '%s' % err
         print(message, file=sys.stderr)
         sys.exit(1)
     except ApplicationException as err:
-        message = 'Application exception: %s' % err
+        message = '%s' % err
         print(message, file=sys.stderr)
         traceback.print_exc(file=sys.stderr)
         sys.exit(2)
     except Exception as err:
-        message = 'Unhandled exception: %s' % err
+        message = '%s' % err
         print(message, file=sys.stderr)
         traceback.print_exc(file=sys.stderr)
-        sys.exit(2)
+        sys.exit(3)
```

Comment:
Opravdu chceme novy exit code? Vracet `2` jako to dela ApplicationException se mi zda ok. Nemuze to neco rozbit v runneru, ktery s temi odpovedmi dale pracuje?

Reply:
Ne. Exit kódy se rozlišují pouze na `0`, `1` a `vše ostatní`. Můžu tam vrátit `2` i prefix `Unhandled exception: `?

Reply:
Ok. Asi bych to teda nechal jak to je. Kdyz se tam vrati prefix tak to potom bude zase vypadat nejak takto `Application exception: Unhandled exception: {real exception message}`.